### ISSUE TITLE
hz serve last minute bugs

### ISSUE THREAD

**Issue by deontologician** [rethinkdb/horizon#683]
- [ ] If schema.toml is not present, `hz serve` bombs out
- [x] `hz serve --help` is missing a space in: `--schema-file=.hz/schema.tomland` in the `--dev` description
- [x] Getting this when creating a new app and using `hz serve`

> App available at http://127.0.0.1:8282
> RethinkDB
>   ├── Admin interface: http://localhost:34085
>   └── Drivers can connect to port 33349
> Ensuring current schema is loaded
> Starting Horizon...
> Initialized new application metadata.
> { ReqlOpFailedError: Table `reproduce_internal.groups` is ambiguous; there are multiple tables with that name in:
> r.db("reproduce_internal").table("groups").get("admin").replace(function(var_2) { return r.branch(var_2.eq(null), {"id": "admin", "rules": {"carte_blanche": {"template": "any()"}}}, var_2.merge({"id": "admin", "rules": {"carte_blanche": r.literal({"template": "any()"})}})); })
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by dalanmiller** [comment]
Is this with using `--dev`? Because I added `--schema-file` to `--dev`.

**Comment by deontologician** [comment]
Yeah, this is with `--dev`

**Comment by dalanmiller** [comment]
Double table creation fixed with 5227bde

**Comment by dalanmiller** [comment]
The first issue in your list still occurs on #696 

```
$ hz serve --dev
App available at http://127.0.0.1:8181
error: rethinkdb stderr: warn: Cache size does not leave much memory for server and query overhead (available memory: 1019 MB).
RethinkDB
   ├── Admin interface: http://localhost:58942
   └── Drivers can connect to port 58941
Ensuring schema ".hz/schema.toml" is applied
serve failed with Error: ENOENT: no such file or directory, open '.hz/schema.toml'
    at Error (native)
```

What should be done if `schema.toml` doesn't exist while using `--dev`? We should probably just skip it in that case.

**Comment by deontologician** [comment]
Fixed
